bin/cli.js

```diff
@@ -4,7 +4,10 @@
 const { gatherTelemetryForUrl, analyzeEmberObject } = require('ember-codemods-telemetry-helpers');
 
 (async () => {
-  await gatherTelemetryForUrl(process.argv[2], analyzeEmberObject);
+  // FIXME: Remove
```

Comment:
These are hacks to make dogfooding work because I never got the telemetry stuff working in my local copy. Was one of the things I wanted to pair on, but this workaround got me far enough.

The FIXME is to remind me to remove it before merge, but I haven't done that yet since I anticipate we'll be making more changes first.